docs/admin/kubeadm-join.md

```diff
@@ -0,0 +1,222 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm join
+notitle: true
+---
+{% capture overview %}
+## Kubeadm join {#cmd-join}
+{% endcapture %}
+
+{% capture body %}
+{% include_relative _kubeadm/kubeadm_join.md %}
+
+### Join workflow 
+`kubeadm init` bootstraps a Kubernetes master node by executing the
+following steps:
+
+The `kubeadm join` bootstraps a Kubernetes worker node and joins it to the cluster. 
+This action consists of the following steps:
+
+1. kubeadm downloads necessary cluster information from the API server.
+   By default, it uses the bootstrap token and the CA key hash to verify the
+   authenticity of that data. The root CA can also be discovered directly via a
+   file or URL.
+
+1. Once the cluster information is known, kubelet can start the TLS bootstrapping
+   process.
+
+   The TLS bootstrap uses the shared token to temporarily authenticate
+   with the Kubernetes Master to submit a certificate signing request (CSR); by
```

Comment:
I don't think "Kubernetes Master" is the proper name of anything. Maybe "Kubernetes master" or "Kubernetes API server".